input.css

```diff
@@ -17,13 +17,17 @@
    .art-s p:nth-of-type(2) {
     @apply h-36 overflow-hidden;
   }
+  .input-sl {
```

Comment:
This class name only appears in the input and output CSS files. Should the class name be added elsewhere so that this style may be applied?

Reply:
we should apply that class name to every input on the web app.

Reply:
I applied this to `input` elements where the style matched, as there were some that had different styles applied and I didn't want to mess that up.